src/css/application.css

```diff
@@ -841,12 +837,18 @@ body {
   text-align: center;
 }
 
-.account-migration__buttons {
+/** @define modal */
```

Comment:
I think we'll want  to standardize/ apply these to the assignment-creator at some point